core-services/sshd-bastion/ppc64le/deployment.yaml

```diff
@@ -14,18 +14,33 @@ items:
       app: prow
       component: sshd
     ports:
-    - name: openstack-auth
-      port: 5000
-      targetPort: openstack-auth
-    - name: api
-      port: 6443
-      targetPort: api
     - name: http
```

Comment:
Remove these if they don't match to anything

Reply:
5000 is the main OpenStack API.  It is the endpoint for authorization.  It will switch to nova or neutron depending on the specific OpenStack call.
6443 will be needed as the OpenShift installation process proceeds.

It was somewhat confusing because I just reordered the list based on port number sorting.
Hope that answers your statement.

Reply:
Uh .... no. The line I am commenting on is at `80` and exposing port `http` ... this port does not exist on the Pod

Reply:
Weird.  I saw a red block and thought you were talking about that.  Sorry.  It should be more obvious that it is right under line 23.  I will fix that as I have yet to find a need for 80 and 443.  I will put them back in when I see something requiring it.